go/private/gazelle.bzl

```diff
@@ -0,0 +1,73 @@
+# Copyright 2016 The Bazel Go Rules Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def _gazelle_script_impl(ctx):
+  script_content = ''
```

Comment:
The initial version was, but then it had a conditional middle part for a while, and then that went away, and I decided to leave it like this in case the conditional part came back again. I think it's probably reasonable stable now, so I will change it back.